CMakeLists.txt

```diff
@@ -1209,6 +1209,15 @@ if(BUILD_SHARED)
           COMPONENT PerfTools)
       endif()
     endforeach()
+
+    if(BUILD_STATIC)
```

Comment:
Here you can do:

```
check_include_files(chrono ZMQ_HAVE_CHRONO)
if(BUILD_STATIC AND ZMQ_HAVE_CHRONO)
```

And it should fix the Windows build

Reply:
I don't think there should be a problem on Windows now since we're using an ifdef.